packages/babel-helper-create-regexp-features-plugin/src/index.js

```diff
@@ -0,0 +1,96 @@
+import rewritePattern from "regexpu-core";
+import {
+  featuresKey,
+  FEATURES,
+  enableFeature,
+  runtimeKey,
+  hasFeature,
+} from "./features";
+import { generateRegexpuOptions } from "./util";
+
+import pkg from "../package.json";
+import { types as t } from "@babel/core";
+import { pullFlag } from "@babel/helper-regex";
+
+// Note: Versions are represented as an integer. e.g. 7.1.5 is represented
+//       as 70000100005. This method is easier than using a semver-parsing
+//       package, but it breaks if we release x.y.z where x, y or z are
+//       greater than 99_999.
+const version = pkg.version.split(".").reduce((v, x) => v * 1e5 + +x, 0);
+const versionKey = "@babel/plugin-regexp-features/version";
+
+export function createRegExpFeaturePlugin({ name, feature, options = {} }) {
+  return {
+    name,
+    pre() {
+      const { file } = this;
+      const features = file.get(featuresKey) || 0;
+      let newFeatures = enableFeature(features, FEATURES[feature]);
+
+      const { useUnicodeFlag, runtime = true } = options;
+      if (useUnicodeFlag === false) {
+        newFeatures = enableFeature(newFeatures, FEATURES.unicodeFlag);
+      }
+      if (newFeatures > features) {
```

Comment:
Could you add a comment like "If we turn on a new bit, it will increase"? It wasn't obvious to me.

Reply:
Or better,
```suggestion
      if (newFeatures !== features) {
```

Reply:
Thanks, I have replaced to `!==`.